core/ir/x86/decode_table.c

```diff
@@ -7056,6 +7060,14 @@ const instr_info_t mod_extensions[][2] = {
     {INVALID, 0x0f0135, catUncategorized, "(bad)", xx, xx, xx, xx, xx, no, x, END_LIST},
     {RM_EXT,  0x0f0175, catUncategorized, "(group 7 mod + rm ext 5)", xx, xx, xx, xx, xx, mrm, x, 5},
   },
+  { /* mod extension 121 */
+    {OP_enqcmds, 0xf338f808, catUncategorized, "enqcmds", GesvS_oq, xx, Moq, xx, xx, mrm, fW6, END_LIST},
```

Comment:
We'd like new entries to have an actual category, even if that's Other.  But although this has further semantics in triggering a device to do something, should we call it a Move?  We can compose these: it could be `catMove|catOther`?

Reply:
These are certainly magic stores. catMove | catOther sgtm.